src/worker/vdb_upload_worker.py

```diff
@@ -179,6 +182,45 @@ def write_embeddings_to_weaviate(text_embeddings_list, vector_db_metadata,  batc
     logging.info(f"Successfully uploaded {len(text_embeddings_list)} vectors to Weaviate")
     return len(text_embeddings_list)
 
+def write_embeddings_to_deeplake(text_embeddings_list, vector_db_metadata, batch_id, job_id):
+    ids = []
+    source_texts = []
+    embeddings = []
+
+    for i, (source_text, embedding) in enumerate(text_embeddings_list):
```

Comment:
Please break this into its own method like how it is for the other vector DBs. That way we can unit test it down the road